wurst/util/PlayerUtils.wurst

```diff
@@ -0,0 +1,104 @@
+package PlayerUtils
+import LinkedList
+
+/**
+*   Credits to TriggerHappy for making PlayerUtils [vJass]
+*    - Link : https://www.hiveworkshop.com/threads/playerutils.278559/
+**/
+
+User array USERS
+
+public function player.toUser() returns User
+    return USERS[this.getId()]
+
+public class User
+    static constant let activeUsers = new LinkedList<User>
+    static constant player local = GetLocalPlayer()
+    static constant int localId = GetPlayerId(User.local)
+
+    protected player _player
+    protected int id
+    protected string originalName
+    protected int index
+    protected bool isPlaying
```

Comment:
Just initialize with true here instead of in the loop

Reply:
Only players that are ACTIVE should be  given isPlaying = true, bots / non-active player slot are supposed to have it as false, maybe it needs to be explicitly set there as an original value. I forget if false is set by default on booleans in Jass or not.

Reply:
Then that should be clarified, and inited nonetheless. Relying on default values isn't nice. It could alsobe replaced by an enum.